cypress/integration/login.ts

```diff
@@ -0,0 +1,20 @@
+import { Web3Provider } from '@ethersproject/providers';
+
+import { TestProvider } from '../util';
+
+context('Coordinape', () => {
+  before(() => {
+    cy.on('window:before:load', win => {
+      const provider = new Web3Provider(
+        new TestProvider('http://localhost:8545')
```

Comment:
How about using process.env.HARDHAT_GANACHE_PORT for this, in preparation for using the testchain?